lib/storage/providers/IDBKeyVal.js

```diff
@@ -102,7 +102,7 @@ const provider = {
      * @param {String} key
      * @return {Promise<*>}
      */
-    getItem: key => get(key, getCustomStore()),
+    getItem: key => get(key, getCustomStore()).then(val => (val != null ? val : null)),
```

Comment:
According to your explanation, shouldn't this check if the value is undefined, and if so, return null. This change looks like the ternary isn't doing anything and is the same thing as returning the value.

Reply:
This will check for null / undefined double `=` and return `null` in both cases, I could change the check to `=== undefined` if you think it is clearer.

Reply:
Yes please. Our styles prefer strict comparison. Could you also please add a comment to the code that the purpose for this logic is to make the storage APIs consistent?

Reply:
Updated